pkg/rpc/request/param.go

```diff
@@ -154,6 +155,27 @@ func (p Param) GetBytesHex() ([]byte, error) {
 	return hex.DecodeString(s)
 }
 
+// GetCosigners returns a slice of transaction.Cosigner with global scope
+// from array of Uint160 stored in the parameter.
+func (p Param) GetCosigners() ([]transaction.Cosigner, error) {
```

Comment:
Not sure where to put this method.

Reply:
That's like a perfect place, three lines to change and we'll have neo-project/neo-modules#260 implemented.